certora/harnesses/AccessControlHarness.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.5.0) (access/AccessControl.sol)
+
+pragma solidity ^0.8.0;
+
+import "../munged/access/AccessControl.sol";
```

Comment:
munged contracts are not part of this PR. What are they? How are they generated? Can they be replaced with the `contracts-exposed` ?

Reply:
munged contracts are created on the fly with a using a `make` file. commands are explained in the file. Im not sure you mean by `contracts-exposed`. We can have a fixed folder of the changed contracts but that would make it difficult to update when changes are made to the original contracts.